rconweb/api/views.py

```diff
@@ -71,27 +72,31 @@ def set_name(request):
     )
 
 
+# TODO: this is not an exposed endpoint
 @csrf_exempt
-@login_required(True)
+@login_required()
 @record_audit
 def set_temp_broadcast(request):
     return set_temp_msg(request, temporary_broadcast, "set_temp_broadcast")
 
 
+# TODO: this is not an exposed endpoint
 @csrf_exempt
-@login_required(True)
+@login_required()
 @record_audit
 def set_temp_welcome(request):
```

Comment:
no permission required here, intended?

Reply:
Yeah, it isn't actually exposed at all so people couldn't use it, so I skipped it to come back to later.